copyright

```diff
@@ -507,6 +525,20 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-3.0
 Comment: Derived from works by Rob Lavinsky (under the same license).
 
+Files:
+ images/star/m-brown-dwarf*
+Copyright: NASA
+License: public-domain
+ From NASA, and therefore in the public domain because they were created by
+ government employees while doing work for the government.
```

Comment:
This image doesn't exist in this PR or the base game.
What does it even mean to be an `m class brown dwarf`?